server/webapp/WEB-INF/rails/config/webpack-base.config.js

```diff
@@ -236,7 +188,61 @@ module.exports = function (env) {
     };
   };
 
-  config.plugins.push(new LicensePlugin());
+  const plugins = [
+    new HappyPack({
+      id:         'jsx',
+      loaders:    [
+        {
+          loader:  'babel-loader',
+          options: {
+            cacheDirectory: path.join(__dirname, '..', 'tmp', 'babel-loader')
+          }
+        }
+      ],
+      threadPool: happyThreadPool
+    }),
+    new HappyPack({
+      id:         'scss',
+      loaders:    [
+        {loader: 'style-loader'},
+        {loader: "css-loader?modules&camelCase&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]"}, // translates CSS into CommonJS
```

Comment:
Use the options hash, like in line 211 and 225?

Reply:
Ok, will do that 👍